CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## Unreleased
 
 - BREAKING: removed Elixir 1.9 and OTP 21 support
+- BREAKING: Rename Shopify API environment variable from `http_timeout` to `recv_timeout`
```

Comment:
🍎 `recv_timeout` seems like a very HTTPoison specific name, are we sure we want to rename to that? I can't think of a better name off the top of my head right now.

Reply:
Thinking of this ticket https://github.com/pixelunion/elixir-shopifyapi/issues/308 and possibly moving to a different HTTP client library.

Reply:
The reason why we picked this name is that there are two variables for HTTPoison `http_timeout` is to specify the time it takes for the connection to timeout before making the connection and `recv_timeout` is the time it takes to wait for the response.  [Docs](https://hexdocs.pm/httpoison/HTTPoison.Request.html)

Reply:
Good catch on the misnaming, what about something like `rest_receive_timeout`?

Reply:
This makes sense. I have changed the name of the variable.